tests/installation/installation_overview.pm

```diff
@@ -120,6 +146,7 @@ sub run {
         set_linux_security_to_none if (is_sle('>=15-SP4') && check_screen("apparmor-not-selected") && !(get_var('PATTERNS') =~ 'default|all|apparmor'));
         ensure_ssh_unblocked;
         $self->check_default_target();
+        $self->disable_secureboot();
```

Comment:
Considering the potential need of enable_secureboot by some test and that this function is directly called in `sub run`, maybe it is more code readable to update logic to something like this?
```
$self->disable_secureboot() if (get_var('UEFI') && get_var('DISABLE_SECUREBOOT')); // remove the condition check from `sub disable_secureboot`
```

Reply:
OK, updated.